pyspecdata/core.py

```diff
@@ -1333,14 +1334,27 @@ def show_array(x, indent=""):
             else:
                 return x
 
+        if self.data.size < 2:
+            val = self.data.item()
+            err = self.get_error()
+            if err is not None:
+                err = err.item()
+                oom_err = int(np.floor(np.log10(err)))  # int takes floor
+                oom_val = int(np.floor(np.log10(val)))  # int takes floor
+                retval = (
+                    "%#0." + str(oom_val - oom_err + 1) + "g ± %#0.2g"
+                ) % (val, err)
```

Comment:
pretty sure this is to get the right sig figs for the value

Reply:
but also this goes with the comment in the example saying we only use 2 sig figs for the error